packages/core/content-releases/admin/src/components/ReleaseModal.tsx

```diff
@@ -30,15 +32,22 @@ export const ReleaseModal = ({
   isLoading = false,
 }: ReleaseModalProps) => {
   const { formatMessage } = useIntl();
+  const { pathname } = useLocation();
+  const isOnUpdate = pathname !== `/plugins/${pluginId}`;
```

Comment:
code style: personally I would check for an exact match that you know. 

```suggestion
  const isCreatingRelease = pathname === `/plugins/${pluginId}`;
```